packages/neos-ui-editors/src/Boolean/index.js

```diff
@@ -25,18 +24,22 @@ const toBoolean = val => {
     return Boolean(val);
 };
 
+const defaultOptions = {
+    disabled: false
+};
+
 const BooleanEditor = props => {
-    const {value, label, identifier, commit} = props;
-    const disabled = $get('options.disabled', props) || false;
+    const {value, label, identifier, commit, options} = props;
+    const finalOptions = Object.assign(defaultOptions, options);
```

Comment:
Hey,

this line mutates the defaultOptions; so correct would be:

`const finalOptions = Object.assign({}, defaultOptions, options)`

Also needs to be fixed in the other places.

All the best,
Sebastian